src/Cryptography/Sodium/Bindings/Signing.hs

```diff
@@ -208,23 +245,19 @@ data CryptoSignState
 -- @since 0.0.1.0
 withSignState :: (Ptr CryptoSignState -> IO a) -> IO a
 withSignState action = do
-  size <- fromIntegral <$> cryptoSignStateBytes
+  let size = (fromIntegral @CSize @Int) cryptoSignStateBytes
```

Comment:
You can also spell this as `let size :: Int = fromIntegral cryptoSignStateBytes`.